server/routes/approvedVisitors.js

```diff
@@ -28,33 +29,32 @@ const createApprovedVisitorsRouter = ({ offenderService }) => {
     };
   };
 
-  router.get(
-    '/',
-    async ({ user, originalUrl: returnUrl, query }, res, next) => {
-      if (config.features.approvedVisitorsFeatureEnabled) {
-        try {
-          const personalisation = user
-            ? await getPersonalisation(user, query)
-            : {};
+  router.get('/', async (req, res, next) => {
+    const { user, originalUrl: returnUrl, query } = req;
```

Comment:
Technically this line can go inside the  `try` too as it's not used anywhere else outside it. But not going to stop merging for that 👍